### DISCUSSION TITLE
Qfieldcloud on android phone behind proxy server

### DISCUSSION THREAD

**Discussion by crniboris** [opengisch/QField#3800] [Q&A]
When  i trying to sync to qfieldcloud from my android phone, which is connected via a wifi network behind a proxy server, I got the following message "Server not found, please check these server URLs". My question is, does qfieldclaud (qfield) on an android phone even work behind a proxy server, and if it does, where do we enter the proxy settings for it.

**Comment by suricactus** [comment]
Double click on the Nyuki logo and enter the new URL there.

**Comment by crniboris** [comment]
Thanks for the answers.
I tried the following links:

https://app.qfield.cloud/a/crniboris
https://app.qfield.cloud/accounts/login/?next=/a/crniboris/
https://app.qfield.cloud/

However, the answer is always the same "Server not found, please check these server URLs".
What am I doing wrong?
What port does QfieldCloud use?

Boris

**Comment by crniboris** [comment]
Interestingly, when I use links:

https://app.qfield.cloud/a/crniboris
https://app.qfield.cloud/accounts/login/?next=/a/crniboris/
https://app.qfield.cloud/

in google search on my phone it works normally and i can open the link.


Boris

**Comment by suricactus** [comment]
I do not follow, please attach a screencast.

**Reply by crniboris** [reply]
Acces Access from Qfield:
![Screenshot_20230113_112121](https://user-images.githubusercontent.com/39115137/212299788-7bd7f45c-1bfd-4c96-b1ab-48c23414dbf4.jpg)

Access from Google search:
![Screenshot_20230113_112857_Chrome](https://user-images.githubusercontent.com/39115137/212299827-2eb84149-c3cd-45b8-ae1f-cb620cee58b7.jpg)
